src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -131,8 +131,14 @@ public function getProperties($class, array $context = []): ?array
     /**
      * {@inheritdoc}
      */
-    public function getTypes($class, $property, array $context = []): ?array
+    public function getTypes($class, $property, array $context = [])
```

Comment:
Shouldn't this return type stay?

Reply:
My bad. Just copy and pasted over different branches.